### ISSUE TITLE
ALT text in images should be searchable

### ISSUE THREAD

**Issue by RMBLRX** [bluesky-social/atproto#1344]
**Is your feature request related to a problem? Please describe.**

ALT text in images is not searchable, which seems a shame because ALT text is a great way to keep image posts elegant and uncluttered.

**Describe the solution you'd like**

I would like to have the text searchable, maybe even specifically searchable.

**Comment by bnewbold** [comment]
This seems pretty reasonable and a positive improvement for everybody.

We might want to double-check with accessibility folks to ensure this doesn't result in unintended consequences. For example, people stuffing SEO keywords in alt text, which would degrade that feature for folks who consume alt text, and also search spam problems. But we could probably ship this and deal with spam issues using regular spam mitigation techniques.

The search code, "palomar", lives over in https://github.com/bluesky-social/indigo, and that is where changes would be made.

**Comment by jcsalterego** [comment]
Cross-referencing: https://github.com/bluesky-social/indigo/pull/248

**Comment by bnewbold** [comment]
this did ship!
